src/store/modules/farmer/types.ts

```diff
@@ -0,0 +1,41 @@
+import { CoreState, CoreData, PartialCoreData } from '../../types';
+
+/*----------------------- Models -----------------------*/
+
+interface BaseFarmer {
+  username: string;
+  firstName: string;
+  middleName: string;
```

Comment:
We don't have `middleName` either.

Reply:
We do have a middlename in the schema don't we? I can make it optional but if it's in the db schema it should be here

Reply:
We do have middlename in schema but not on agriTrader, so yeah I think optional is better.

Reply:
So it's not a required field for them to provide? @enochtsang can you confirm?